xml/System.Security.Cryptography/AsymmetricAlgorithm.xml

```diff
@@ -459,11 +459,27 @@
         <Parameter Name="pbeParameters" Type="System.Security.Cryptography.PbeParameters" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="passwordBytes">To be added.</param>
-        <param name="pbeParameters">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="passwordBytes">The bytes to use as a password when encrypting the key material.</param>
+        <param name="pbeParameters">The password-based encryption (PBE) parameters to use when encrypting the key material.</param>
+        <summary>Exports the current key in the PKCS#8 EncryptedPrivateKeyInfo format.</summary>
+        <returns>A byte array containing the PKCS#8 EncryptedPrivateKeyInfo representation of this key.</returns>
         <remarks>To be added.</remarks>
+        <remarks>
```

Comment:
These remarks aren't appearing in the build, and I have no idea why

Reply:
Looks like I didn't delete the `<remarks>To be added.</remarks>` when pasting from another version, which probably did it.

Reply:
Yes. I didn't notice the duplicated \<remarks> tags.